baselines/webworker.generated.d.ts

```diff
@@ -450,6 +450,15 @@ declare var FileReader: {
     new(): FileReader;
 }
 
+interface FormData {
+    append(name: string, value: string | Blob, fileName?: string): void;
+}
+
+declare var FormData: {
+    prototype: FormData;
+    new (form?: HTMLFormElement): FormData;
```

Comment:
Tis is failing the CI tests:

> generated/webworker.generated.d.ts(459,17): error TS2304: Cannot find name 'HTMLFormElement'.

you will need to update this for WebWorkers only to be `string` i presume.